src/injectIntoComponent.js

```diff
@@ -0,0 +1,46 @@
+import { PropTypes } from 'react';
+import Injector from './Injector';
+/**
+ * Creates an extended component that will use injector-js DI to inject into the context.
+ * @param  {React.Component} Component The Component To Extend
+ * @return {React.Component}           Extended Component
+ */
+export default function injectIntoComponent(Component) {
```

Comment:
I don't think it will work too well:

```
S 1.9.8 (Mac OS X 0.0.0) React Integration Inject into Component should inject simple values into the component class FAILED
	Invariant Violation: InjectorComponent.getChildContext(): childContextTypes must be defined in order to use getChildContext().
	    at invariant (/Users/steven/Sites/injector.js/test.js:1189 <- webpack:///~/fbjs/lib/invariant.js:46:0)
	    at /Users/steven/Sites/injector.js/test.js:7490 <- webpack:///~/react/lib/ReactCompositeComponent.js:328:0
	    at /Users/steven/Sites/injector.js/test.js:7387 <- webpack:///~/react/lib/ReactCompositeComponent.js:225:0
	    at ReactCompositeComponent_mountComponent (/Users/steven/Sites/injector.js/test.js:1500 <- webpack:///~/react/lib/ReactPerf.js:66:0)
	    at /Users/steven/Sites/injector.js/test.js:20216 <- webpack:///~/react/lib/ReactTestUtils.js:362:0
	    at /Users/steven/Sites/injector.js/test.js:20198 <- webpack:///~/react/lib/ReactTestUtils.js:344:0
	    at /Users/steven/Sites/injector.js/test.js:19811 <- webpack:///spec/injectIntoComponentSpec.js:21:8
```